test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/function/SupplierTest.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.javalib.util.function
+
+import java.util.function.Supplier
+
+import org.junit.Assert._
+import org.junit.Test
+
+class SupplierTest {
```

Comment:
I'm not sure this needs a test beyond something that simply checks it compiles. (There is value in that, to ensure the interfaces are correct, but actually calling something seems moot).